life_expectancy/tests/test_data_cleaning.py

```diff
@@ -0,0 +1,23 @@
+"""Tests for the cleaning module"""
+import pandas as pd
+
+from life_expectancy.data_cleaning import clean_data
+from . import FIXTURES_DIR
+
+
+def test_clean_data(
+    eu_life_expectancy_loaded_dataframe_test,
+    pt_life_expectancy_expected_test
+    ):
+    """Test the `clean_data` function and compare the output to the expected output"""
+
+    cleaned_dataframe_test = clean_data(eu_life_expectancy_loaded_dataframe_test)
+
+    print(cleaned_dataframe_test.shape)
+    print(cleaned_dataframe_test.index)
+    print(pt_life_expectancy_expected_test.shape)
```

Comment:
Might be nice to compare the shape and index of both in an assert. Doing the comparison automatically instead of checking manually. Would be useful if the pandas assert_frame_equal fails to give a hint to what might be the problem.

Reply:
this was extra code that I forgot to delete